lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -217,7 +217,9 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                 % endif
             % endif
 
-            % if not entitlement:
+            % if entitlement:
+              <%include file='_dashboard_entitlement_actions.html' args='course_overview=course_overview,entitlement=entitlement,dashboard_index=dashboard_index'/>
+            % else:
```

Comment:
Is the `is_course_blocked` state relevant for entitlements? If so, we may need to make some changes around line 326.

Reply:
also need to check if the `can_unenroll` state applies to entitlements.